src/index.html

```diff
@@ -7,8 +7,38 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700;900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="flexbox">
```

Comment:
```suggestion
    <header class="header">
```
It's bad style name.
![image](https://user-images.githubusercontent.com/102293862/202490068-353c3891-1b58-472d-935f-b4f9249b148a.png)